skrf/network.py

```diff
@@ -3817,6 +3817,23 @@ def nonreciprocity(self, m: int, n: int, normalize: bool = False) -> Network:
         else:
             return (forward - reverse)
 
+    def s_error(
+        self,
+        ntwk: Network,
+        error_function: Literal[
```

Comment:
you can define the literal somewhere in constants if you need it more than once.